src/transformer/utils/get_activity/course_module.php

```diff
@@ -47,10 +47,14 @@ function course_module(array $config, \stdClass $course, int $cmid, string $xapi
     $courselang = utils\get_course_lang($course);
     $instancename = property_exists($instance, 'name') ? $instance->name : $module->name;
 
+    $activitytype = utils\get_module_activity_type($module->name);
```

Comment:
This is basically the biggest change here, xAPI type is decided by a single table, not whatever the caller specifies.